docs/en/enterprise-edition/content-collections/dashboards/dashboards-discovery-exposure-management.adoc

```diff
@@ -65,17 +83,18 @@ While you *Inspect Unmanaged Assets*, you can choose to snooze your known intern
 
 Once snoozed, Prisma Cloud will provide the flexibility to view your *Active* assets separately from the *Snoozed* assets on the *Inventory > Unmanaged Assets* page, simplifying asset management.
 
-
-* *Unmanaged Exposed Assets by Country*
+[#cdem-widget-4]
+==== Unmanaged Exposed Assets by Country
 
 image::dashboards/cdem-unmanaged-assets-map.gif[]
 
 Displays a geographical view of your exposed assets on a map. Each pointer on the map represents the exposed asset's location. Once you identify a region to secure, select the appropriate button to *Convert* or *Inspect* the unmanaged assets for better security.
 
-The *Convert* button is enabled only for accounts associated with the parent account or organizational unit (OU) accounts on Prisma Cloud. In cases where such mapping is not feasible, you can use the *Inspect* button to identify issues related to exposed assets and *Send email* to your inbox containing the summary of the exposed assets. 
+The *Convert* button is enabled only for accounts associated with the parent account or organizational unit (OU) accounts on Prisma Cloud. Selecting the *Convert* button will onboard the assets to Prisma Cloud. Once the unmanaged assets convert to managed assets, Prisma Cloud will monitor these assets for vulnerabilities, misconfigurations, compliance issues, and so on.
 
+In cases where such mapping is not feasible, you can use the *Inspect* button to identify issues related to exposed assets and *Send email* to your inbox containing the summary of the exposed assets. Selecting the *Inspect* button opens a side panel. The panel includes a comprehensive list of exposed assets, risk categories, and associated cloud providers. Further exploration of each asset provides detailed information such as asset *Overview*, *Exposed Services*, *Vulnerabilities*, *Findings*, and *Network Flow Log* visualizations.
```

Comment:
Suggest the following rewrite:

In cases where such mapping is not feasible, use the *Inspect* button to identify issues related to exposed assets and *Send email* to your inbox containing the summary of the exposed assets. Select *Inspect* to view a side panel, with a comprehensive list of exposed assets, risk categories, and associated cloud providers. Further exploration of each asset provides detailed information such as asset *Overview*, *Exposed Services*, *Vulnerabilities*, *Findings*, and *Network Flow Log* visualizations.